File: polocloud-suite/src/main/java/dev/httpmarco/polocloud/suite/commands/impl/ClearCommand.java
package dev.httpmarco.polocloud.suite.commands.impl;

import dev.httpmarco.polocloud.suite.commands.Command;
import dev.httpmarco.polocloud.suite.utils.ConsoleActions;

public final class ClearCommand extends Command {

    public ClearCommand() {
        super("clear", "Clear the console terminal", "cc");

        defaultExecution(commandContext -> ConsoleActions.clearScreen());
    }
}

File: agent/src/main/java/dev/httpmarco/polocloud/agent/runtime/local/terminal/commands/impl/ClearCommand.kt
package dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.impl

import dev.httpmarco.polocloud.agent.runtime.local.terminal.Jline3Terminal
import dev.httpmarco.polocloud.agent.runtime.local.terminal.commands.Command

class ClearCommand(private val terminal : Jline3Terminal) : Command("clear", "Clears the terminal screen", "cc") {

    init {
        defaultExecution {
            terminal.clearScreen()
        }
    }
}